src/main/config.js

```diff
@@ -9,6 +9,7 @@ export const defaultWinOptions = {
   minWidth: 450,
   minHeight: 220,
   webPreferences: {
+    experimentalFeatures: true,
```

Comment:
What do we need `experimentalFeatures`?

Reply:
... I just use it for test some experimental features of Chrome, I'll remove it.